src/app.py

```diff
@@ -78,90 +81,312 @@ def index():
 """
 Show the current VERSION and BUILD, as well as the status of MySQL connection.
 
+Parameters
+----------
+None
+
 Returns
 -------
-json
-    A json containing the status details
+HTTP 200 Response with Content-Type application/json
+The JSON body is a dictionary with the status details.
 """
 # Status of MySQL connection
 @app.route('/status', methods=['GET'])
 def status():
-    global worker
+    global ukv_worker
 
-    response_data = {
-        # Use strip() to remove leading and trailing spaces, newlines, and tabs
+    # Use strip() to remove leading and trailing spaces, newlines, and tabs
+    status_data = {
         'version': (Path(__file__).absolute().parent.parent / 'VERSION').read_text().strip(),
         'build': (Path(__file__).absolute().parent.parent / 'BUILD').read_text().strip(),
-        'mysql_connection': False
+        'mysql_connection': ukv_worker.test_connection()
     }
-
-    dbcheck = worker.testConnection()
-    if dbcheck:
-        response_data['mysql_connection'] = True
-
-    return jsonify(response_data)
+    return jsonify(status_data)
 
 """
 An endpoint to create or update a key/value pair for the authenticated user.
 The value is valid JSON attached to the HTTP Request.
-On success returns an HTTP 200 Response.
 
 Parameters
 ----------
-key : The key to store the value under for the authenticated user, as specified on the API endpoint 
+key : The key to store the value under for the authenticated user, as specified on the API endpoint. The
+      key cannot contain any whitespace, the characters %, _, *, #,',",`, or the character sequences /*, */, or --.
```

Comment:
This instruction can be used in the 4xx client errors so the users know how to format the key properly.

Reply:
Should already show up in error messages under those circumstance per [these lines](https://github.com/x-atlas-consortia/user-key-value-api/blob/480767588c541693111c410bd0789afa81c96def/src/ukv_worker.py#L118-L125).